docs/environments/gke.md

```diff
@@ -63,3 +62,74 @@ kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=clu
 
 Once all pods are running you can return to the [User Guide](../userguide.md) to deploy the
 KubeFed control-plane.
+
+## Additional firewall rule for kubefed-admission-webhook
+
+Admission control in GKE requires an additional VPC network firewall rule.
+
+### Background
+
+The GCP Kubernetes control plane is deployed in a Google-managed account.
+
+During cluster provisioning, GKE automatically creates:
+1. A network peering connection to allow the master to communicate to the node pools.
+1. A firewall rule to allow ports 443 and 10250 from the master to the node pools.
+
+The GKE API Server operates with [`--enable-aggregator-routing=true`](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L1586). This means that the Service IPs are internally translated to endpoint IPs and routed to the relevant pods.
```

Comment:
Is there an official GKE doc that describes this flag being set by default which could be linked instead? If not, the link provided here to a specific line will undoubtedly change as it's not a permalink.

Reply:
Very good point. Unfortunately there isn't - as far as I know. However, I would link the [kubernetes issue](https://github.com/kubernetes/kubernetes/issues/79739) I opened with them. They promised to create some documentation to inform GCP users regarding this behaviour.